nginx/sites-enabled/default

```diff
@@ -11,6 +11,12 @@ server {
 
 	client_max_body_size 500M;
 
+    proxy_read_timeout 600s;
+    proxy_connect_timeout 600s;
+    proxy_send_timeout 600s;
+    fastcgi_send_timeout 600s;
+    fastcgi_read_timeout 600s;
```

Comment:
indentation seems a bit off?

And also here the timeout might be really high.
I think 3 or 5 minutes are also fine

Reply:
nevermind. we need 10 minutes for big plugins like woocommerce